lib/kino/explorer.ex

```diff
@@ -46,24 +46,50 @@ defmodule Kino.Explorer do
 
   @impl true
   def get_data(rows_spec, state) do
-    records = get_records(state.df, rows_spec)
+    {records, total_rows, summaries} = get_records(state.df, rows_spec)
+    columns = Enum.map(state.columns, &%{&1 | summary: summaries[&1.key]})
     rows = Enum.map(records, &record_to_row/1)
-    {:ok, %{columns: state.columns, rows: rows, total_rows: state.total_rows}, state}
+    {:ok, %{columns: columns, rows: rows, total_rows: total_rows}, state}
   end
 
   defp get_records(df, rows_spec) do
     df =
-      if order_by = rows_spec[:order_by] do
-        Explorer.DataFrame.arrange_with(df, &[{rows_spec.order, &1[order_by]}])
-      else
-        df
-      end
+      df
+      |> order_by(rows_spec.order, rows_spec[:order_by])
+      |> filter_by(rows_spec[:filters])
 
+    total_rows = Explorer.DataFrame.n_rows(df)
+    summaries = if total_rows > 0, do: summaries(df)
     df = Explorer.DataFrame.slice(df, rows_spec.offset, rows_spec.limit)
-
     {col_names, lists} = df |> Explorer.DataFrame.to_columns() |> Enum.unzip()
+    records = Enum.zip_with(lists, fn row -> Enum.zip(col_names, row) end)
```

Comment:
I think something like this will be slightly more efficient:

```suggestion
    columns = Explorer.DataFrame.to_columns(df)
    col_names = Map.keys(columns)
    records = Enum.zip_with(columns, fn {_, row} -> Enum.zip(col_names, row) end)
```

But I also wonder if we should change `Kino.Table` to send information as a columns instead of rows (or perhaps both). It would definitely reduce the amount of data encoded over the wire.

WDYT @jonatanklosko? Should we open up a separate issue for that? @cristineguadelupe, how does Glide usually load data? As rows or as columns?

Reply:
As cells!

Reply:
What do you mean? You set the data of each cell? Or do you give it a JSON-like structure? Can you point me to the code please? :)

Reply:
https://github.com/livebook-dev/kino/blob/main/assets/data_table/main.js#L252-L267

Reply:
Got it!

Reply:
Moved the discussion here: https://github.com/livebook-dev/kino/issues/232